Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -205,12 +213,19 @@ def get_ticket_human_readable(tickets, ticket_type: str) -> list:
         priority = ticket.get('priority', '')
         if priority:
             hr['Priority'] = TICKET_PRIORITY.get(priority, priority)
+
         state = ticket.get('state', '')
         if state:
             mapped_state = state
             if ticket_type in TICKET_STATES:
                 mapped_state = TICKET_STATES[ticket_type].get(state, mapped_state)
             hr['State'] = mapped_state
+        approval = ticket.get('approval', '')
+        if approval:
+            mapped_approval = approval
+            if ticket_type in TICKET_APPROVAL:
+                mapped_approval = TICKET_APPROVAL[ticket_type].get(ticket.get('approval'), mapped_approval)
+                hr['Approval'] = mapped_approval
```

Comment:
this should be indented to the left like in state I think

Reply:
If it does, than it is being added to all the other ticket types when CRUDing them, so keeping this indented.